bin/cleanup/cleanup.py

```diff
@@ -507,7 +528,7 @@ def cleanup_server(
 
 
 def delete_leaked_td_resources(
-    dry_run, td_resource_rules, project, network, resources
+    dry_run, td_resource_rules, project, network, resources, enable_dualstack
```

Comment:
Unlike other places I suggested to set the default value, this one actually has a call with `enable_dualstack` unfilled, line 759 (below `if cleanup_legacy:`)

```suggestion
    dry_run,
    td_resource_rules,
    project,
    network,
    resources,
    enable_dualstack: bool = False,
```

Reply:
Yep, I see this in the log

```py
Traceback (most recent call last):
  File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/tmp/tmp.oMTIo1ZPtP/psm-interop/bin/cleanup/cleanup.py", line 814, in <module>
    app.run(main)
  File "/tmp/tmp.oMTIo1ZPtP/psm-interop/venv/lib/python3.10/site-packages/absl/app.py", line 308, in run
    _run_main(main, args)
  File "/tmp/tmp.oMTIo1ZPtP/psm-interop/venv/lib/python3.10/site-packages/absl/app.py", line 254, in _run_main
    sys.exit(main(argv))
  File "/tmp/tmp.oMTIo1ZPtP/psm-interop/bin/cleanup/cleanup.py", line 785, in main
    find_and_remove_leaked_td_resources(
  File "/tmp/tmp.oMTIo1ZPtP/psm-interop/bin/cleanup/cleanup.py", line 759, in find_and_remove_leaked_td_resources
    delete_leaked_td_resources(
TypeError: delete_leaked_td_resources() missing 1 required positional argument: 'enable_dualstack'
```

Reply:
oh ok. Thanks for noticing. Now that we have default value and we are using td_no_legacy mode it should not happen